wagtailpolls/templates/wagtailpolls/poll_list.html

```diff
@@ -25,7 +25,7 @@
 					<td class='created_on'>
 						{% if poll.date_created %}
 						<div class="human-readable-date" title="{{ poll.date_created|date:"d M Y H:i" }}">
```

Comment:
There are a few formats that the `|date` filter understands that produce locale appropriate dates. Under the big table of format options, [the date docs](https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#date) have more info. In this case, something like `{{ poll.date_created|date:"SHORT_DATETIME_FORMAT" }}` would be appropriate.